.github/workflows/destroy_aks.yml

```diff
@@ -0,0 +1,46 @@
+name: Destroy Review Instance
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  destroy:
+    name: Destroy
+    environment:
+       name: review_aks
+    runs-on: ubuntu-latest
+    concurrency: review_aks${{github.event.number}}
+
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set Environment variables
+        id: set_env_var
+        shell: bash
+        run: |
+          tf_vars_file=terraform/aks/config/${{ inputs.environment }}.tfvars.json
+          echo " tf_vars_file is $tf_vars_file"
+          terraform_version=$(awk '/{/{f=/^terraform/;next}f' terraform/aks/terraform.tf | grep -o [0-9\.]*)
+          echo "cluster=$(jq -r '.cluster' ${tf_vars_file})" >> $GITHUB_ENV
```

Comment:
"cluster" and "aks_app_environment" are not used anywhere and can be removed

Reply:
done